resources/install.rb

```diff
@@ -44,6 +44,7 @@
 property :feature, [Array, String], default: %w(SQLENGINE REPLICATION SNAC_SDK)
 property :install_dir, String, default: 'C:\Program Files\Microsoft SQL Server'
 property :instance_dir, String, default: 'C:\Program Files\Microsoft SQL Server'
+property :shared_wow_dir, String, default: 'C:\Program Files (x86)\Microsoft SQL Server'
```

Comment:
Can you just make the default use the same existing behavior

```suggestion
property :shared_wow_dir, String, default: lazy { install_dir.gsub(/Program Files/, 'Program Files (x86)') }
```

Reply:
Commit done